### PR TITLE
ignacio-python-lab_03b

### PR DESCRIPTION
**Pull request by RemmyI**

### CODE REVIEW

File: code/ignacio/lab_03b_mad_libs.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+while True:
+    num = input("Enter number, or 'done': ")
+
+    if num == "done":
+        print(f"Average:  ")
+        break
+
+
+    nums.append(num)
+    print(nums)
```
Comment (cdmichaelb) [inline code/ignacio/lab_03b_mad_libs.py]: This is just printing the numbers, not the average.

Also, this is titled mad libs but appears to be the average numbers lab.

File: code/ignacio/lab_5a_palindrome.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# import string
+
+
```
Comment (cdmichaelb) [inline code/ignacio/lab_5a_palindrome.py:3]: try again with these two inputs:

"anagram" and "nag a ram"
"anagram" and "Anagram"

Both should be anagrams. Check the assignment for further missing steps.

File: code/ignacio/lab_5a_palindrome.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+#     print(f'{word_1} and {word_2} are not anagrams')
+# --------------------------------------------------
+
+subject = input('enter a word: ')
+
+def check_palindrome(pal_check):
+    reverse_check = list(pal_check)
+    reverse_check.reverse()
+    reverse_check = ''.join(reverse_check)
+    print(reverse_check)
```
Comment (cdmichaelb) [inline code/ignacio/lab_5a_palindrome.py:30]: Shouldn't print inside of a function.

Also, there is no evaluation to see if the word is a palindrome.

**Review by cdmichaelb — CHANGES_REQUESTED** [review]
Need changes to complete assignments.

**Review by ttajohnson — APPROVED** [review]

**Comment by cdmichaelb** [comment]
Submitted in #57

**Pull request closed without merging**